propagation_ls.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/base64"
 
 	"github.com/golang/protobuf/proto"
-	lightstep "github.com/lightstep/lightstep-tracer-go/lightsteppb"
+	lightstep "github.com/lightstep/api-public/go/lightstep"
```

Comment:
Does this need to be updated to `lightstep-tracer-protos`?

Reply:
Oops. Good catch!